src/bin/pgcopydb/ld_stream.h

```diff
@@ -275,6 +275,26 @@ typedef enum
 } LogicalStreamMode;
 
 
+/*
+ * Fully Qualified Postgres column name: "nspname"."relname"."attname".
+ * We need to account for the dots hence add 2 more bytes.
+ */
+typedef char FQColumnName[(PG_NAMEDATALEN * 3) + 2];
```

Comment:
I think we do not need that. See the docs for UTHASH support for both [compound keys](https://troydhanson.github.io/uthash/userguide.html#_compound_keys) and also for [structure keys](https://troydhanson.github.io/uthash/userguide.html#_structure_keys).

Reply:
Done.